### PR TITLE
Allow adjustment of Jicofo and jitsi-videobridge log verbosity

### PR DESCRIPTION
**Pull request by ageis**

### CODE REVIEW

**Comment by conorsch** [comment]
Where are the docs for the log levels? Let's add a comment in the default vars file, pointing to a URL or other resource for reference.

**Comment by ageis** [comment]
@conorsch So these are just Java log levels, so the appropriate doc is: https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html Jitsi-Meet doesn't appear to have specific docs for their logging facilities.

File: tasks/jicofo.yml [hunk 1]
```diff
@@ -16,3 +16,11 @@
     group: root
     mode: "0644"
   notify: restart jitsi-videobridge
+
+- name: Set Jicofo log level.
+  lineinfile:
+    dest: /etc/jitsi/jicofo/logging.properties
+    regexp: "^.level"
```
Comment (conorsch) [inline tasks/jicofo.yml]: Should be `"^\.level"` (escape the period).

**Comment by conorsch** [comment]
Great, let's still link to it. We're making the option configurable and it's good to explain why:

```
# The Jitsi components use the standard Java log levels, see:
# https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html
# When using log aggregation for jitsi-meet components, set to "WARNING".
jitsi_meet_jicofo_loglevel: INFO
```

**Comment by conorsch** [comment]
Tests fail with the new change—working on it. Will fix, update tests, and append to this branch prior to merge.

**Comment by conorsch** [comment]
Oops, left this hanging. Will rerun test suite and report back.

**Comment by conorsch** [comment]
Looking good, merging.

**Pull request merged**